doc/source/docs/user_guide/io.rst

```diff
@@ -143,18 +143,17 @@ or a slice object.
 Field/column filters
 ^^^^^^^^^^^^^^^^^^^^
 
-Load in a subset of fields from the file:
-
-.. note:: Requires Fiona 1.9+
+Load in a subset of fields from the file using the ``columns`` keyword:
 
 .. code-block:: python
 
     gdf = geopandas.read_file(
         geodatasets.get_path("geoda.nyc"),
-        include_fields=["name", "rent2008", "kids2000"],
+        columns=["name", "rent2008", "kids2000"],
     )
 
-.. note:: Requires Fiona 1.8+
+This requires pyogrio or Fiona 1.9+. In addition, it's also possible to specify
+which columns _not_ to read:
```

Comment:
Do we want to keep this `ignore_fields` example? 
If so, I probably should remove the warning about it when using pyogrio. However, for pyogrio it does require an additional file access for `pyogrio.read_info`, so I do think it is recommended to use `columns`

Reply:
Removed this for now.